rest/service/src/v0/children.rs

```diff
@@ -0,0 +1,185 @@
+use super::*;
+
```

Comment:
I see a lot of of fn some_rest_method() -> match xxx.await { Ok(o) => Http.ok().json(o), Err(e) => Err(rest::from(e).into) }

or something like that at least. I cant help but wonder: can this be made more ergonomic?

Reply:
Yeah, I know... The `?` probably doesn't help here because we return a responder trait, not an error. I'll see what I can do...

Reply:
Yeah this works, I'll change all of the handlers from impl Responder to Result<...>

Reply:
Hmm the return type doesn't quite work after all... back to the drawing board...